documentation/user-consent.md

```diff
@@ -48,122 +43,269 @@ The document details aspects regarding the consent management, which includes fo
+>[TO BE EDITED/COMPLETED] - The base standard is OpenID Connect (OIDC). <br>User authentication: Verify that the user is who they say they are. For example: IP/Network authentication, SMS OTP, User/Password, Push to operator application... it depends on the type of authorization (grant type) chosen according to the application use cases, integration model and requirements (B2B/B2C, via aggregator or not...).<br>User authorization: Verify access to the requested resource (OpenID Connect scope, purpose) through policies and rules... user consent is checked at this point as a pre-requisite before issuing an access token.
 
 ### Authentication mechanism/s
 
 >[TO BE EDITED/COMPLETED] - Authentication mechanism/s to be considered... E.g.: IP/Network authentication, SMS OTP, User/password, Push to operator application​...
 
 ### Consent capture and storage
 
->[TO BE EDITED/COMPLETED] - Consent capture (by operator vs by API Client a.k.a delegated) and storage (in operator vs in a third party).
+>[TO BE EDITED/COMPLETED] - Consent capture by operator and storage in operator (consent master).
 
 ### Authorization flows / grant types
 
->[TO BE EDITED/COMPLETED] - Authorization flows / grant types to be considered... E.g.: Authorization code flow, JWT-bearer flow, CIBA flow, Client credentials flow. For each API it should be possible to categorize it depending on the use case, integration/GTM model, personal information processing requirements, etc... and define the grant type/s to be used for each API.
-
-#### Authorization code flow
+#### Authorization code flow (Frontend flow)
 
->[TO BE EDITED/COMPLETED] - Flow to obtain an access token to consume network API...
+```mermaid
+sequenceDiagram
+autonumber
+title Consume a CAMARA API - Authorization Code Grant (FrontEnd)
+participant FE as Device App
+participant BE as Invoker (App Backend/Aggregator)
+box Operator
+  participant ExpO as OGW Platform<br>@Operator
+  participant Consent as Consent Master<br>@Operator
+end
 
-#### JWT-bearer flow
+Note over FE,BE: Use Feature needing<br>Operator Capability  
+BE->>FE: Auth Needed - redirect <br>/authorize?response_type=code&client_id=coolApp<br>&purpose=purpose&redirect_uri=invoker_callback...
+FE->>+FE: Browser /<br> Embedded Browser
+alt Standard OIDC Auth Code Flow between Invoker and OGW Platform
+  FE-->>ExpO: GET /authorize?response_type=code&client_id=coolApp&purpose=purpose&redirect_uri=invoker_callback...
+  Note over ExpO: OGW Platform Applies<br>Network Based Authentication (amr=nba/mnba)
```

Comment:
```suggestion
  Note over ExpO: API Gateway Platform Applies<br>Network Based Authentication (amr=nba/mnba)
```
Is _Network Based Authentication (amr=nba/mnba)_ defined in OIDC specs?

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/54/commits/c7f9715fa8c597e4c9f94e8f0bbf3aecc501ead5.